schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals(
+  id INT PRIMARY KEY, 
+  name CHAR(30), 
+  date_of_birth DATE, 
+  escape_attempts INT, 
+  neutered BOOL, 
```

Comment:
* Kindly add varchar for all your columns as it determines the length of characters this is best practice 👍